### ISSUE TITLE
`TimeTest::testGetAge` fails

### ISSUE THREAD

**Issue by kenjis** [codeigniter4/CodeIgniter4#6818]
```
There was 1 failure:

1) CodeIgniter\I18n\TimeTest::testGetAge
Failed asserting that 4 is identical to 5.

/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/I18n/TimeTest.php:399
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/3399510319/jobs/5653309525

> 5 Nov 2017 - Daylight Saving Time Ended
> When local daylight time was about to reach
> Sunday, 5 November 2017, 02:00:00 clocks were turned backward 1 hour to
> Sunday, 5 November 2017, 01:00:00 local standard time instead.
https://www.timeanddate.com/time/change/usa/chicago?year=2017

> 6 Nov 2022 - Daylight Saving Time Ends
> When local daylight time is about to reach
Sunday, 6 November 2022, 02:00:00 clocks are turned backward 1 hour to
Sunday, 6 November 2022, 01:00:00 local standard time instead.
https://www.timeanddate.com/time/change/usa/chicago?year=2022

`$testTime`: 2022-11-05 10:14:35.000000 UTC
`$ourTime`: 2017-11-05 11:14:35.000000 UTC

https://github.com/codeigniter4/CodeIgniter4/blob/5c2d10be45193f7e1b6579a0ff338e9f821c74cb/system/I18n/Time.php#L1056-L1062

**Comment by kenjis** [comment]
```php
        d(Time::parse('5 years ago')->format('c')); // string (25) "2017-11-05T07:17:03-06:00"
        dd(Time::now()->format('c'));               // string (25) "2022-11-05T07:17:03-05:00"
```

**Comment by kenjis** [comment]
I do not know if this is a bug or not. But clearly it is not intuitive that this test fails.
https://github.com/codeigniter4/CodeIgniter4/blob/5c2d10be45193f7e1b6579a0ff338e9f821c74cb/tests/system/I18n/TimeTest.php#L395-L401

**Comment by kenjis** [comment]
@totoprayogo1916 Yes, this is because of Daylight Saving Time.

**Comment by paulbalandan** [comment]
Can we change the default timezone of tests to use UTC? At least UTC is not affected by DST.
> The switch to daylight saving time does not affect UTC. It refers to time on the zero or Greenwich meridian, which is not adjusted to reflect changes either to or from Daylight Saving Time.
https://www.nhc.noaa.gov/aboututc.shtml#:~:text=The%20switch%20to%20daylight%20saving,or%20from%20Daylight%20Saving%20Time.

**Comment by kenjis** [comment]
Daylight saving time, it's horrible.

**Comment by MGatner** [comment]
US is considering abolishing the shifts: https://en.m.wikipedia.org/wiki/Sunshine_Protection_Act

**Comment by kenjis** [comment]
@paulbalandan See #6826

**Comment by paulbalandan** [comment]
Fixed by #6826
